charms/layers/jupyter-notebook/templates/upstart.conf

```diff
@@ -0,0 +1,11 @@
+#!upstart
+description "jupyter"
+
+start on startup
+stop on shutdown
+
+respawn
+
+script
+jupyter notebook --no-browser
```

Comment:
@sebastienpattyn93 If jupyter runs as root, `~/.jupyter/jupyter_notebook_config.py` will point to `/root`, no?